virt/virt-2-6-release-notes.adoc

```diff
@@ -95,6 +97,31 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //BZ-1855182 (https://bugzilla.redhat.com/show_bug.cgi?id=1855182): [Storage] Clone could not be continued after virtctl stop the vm if the clone dv have been created for more than 3 minutes
 
+* The Containerized Data Importer (CDI) and KubeVirt depend on QEMU which does not support NFS version 3. Therefore, only NFS version 4 is supported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1892445[*BZ#1892445*])
+
+* The name of the Fedora PVC in the `openshift-virtualization-os-images` namespace is `fedora`, instead of `fedora32`. If you populated the `fedora32` PVC in {VirtProductname} 2.5 or earlier, the virtual machine does not appear in the web console and you cannot use it to clone another virtual machine. As a workaround, upload a Fedora image by naming the PVC `fedora`, instead of `fedora32`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913352[*BZ#1913352*])
+
+* When creating a HPP boot source, the data volume is `pending` with a `WaitForFirstConsumer` status if a user creates the boot source using any method except the *Upload local file (creates PVC)* option. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929177[*BZ#1929177*])
++
+As a workaround, in the *Storage* > *Persistent Volume Claims* web console screen, edit the YAML of the underlying PVC of the data volume to add the `cdi.kubevirt.io/storage.bind.immediate.requested: "true"` `annotation`:
```

Comment:
Can you edit the YAML directly in the *Storage* > *Persistent Volume Claims* screen?

Reply:
Resolved this with Shikha. This feature is UI only for 2.6, just FYI